README.md

```diff
@@ -8,4 +8,11 @@ This is a 3d printable gas mask design.  It's printed in 3 parts - the main mask
 
 The filter material is activated charcoal - you can buy this at any pet store.  Put a layer of cotton batting in the bottom of the canister, then fill with activated charcoal, then put another layer of cotton on top, and screw the canister into the mask.
 
+## Safety Warnings
+1. Print Material: PLA can be brittle and sharp in the event that it breaks. It is recommended to use PETG or Nylon if possible, especially if the risks of PLA shattering could be an issue in the environment you plan on using it or if you expect the mask to be subjected to stresses or impacts.
+2. Face Gasket: When using a hard plastic material, it is extremely unlikely that a proper seal will form between the mask and the user's face. Some form of gasket or appropriate foam should be added around the edge of the mask to ensure an airtight seal.
+3. Fit / Seal / Leak Testing: It is highly recommended to test the mask for leaks and fit quality.
+4. Exhale / Vent Valve: The amount of dead space inside the mask and filters can lead to a risk of asphyxiation. All masks in this repository have provisions for installing an exhale valve. Please be sure to assemble and install one in every mask.
```

Comment:
I like this suggestion, but since we don't yet have a working valve design, the wording might be a bit confusing.  Can you update this to mention that we're still working on a good valve design, and that you shouldn't use this mask in a real world situation until we have a working valve?